db/migrate/20170912194047_add_col_to_order.rb

```diff
@@ -0,0 +1,21 @@
+class AddColToOrder < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
Shouldn't it contain a colomn for selecting pick up or delivery?

Reply:
Yes, it does. It is the: delivery_method, :string

Reply:
Good spot. This might be extracted to a separate feature if not fixed now

Reply:
I hought the customer should tick a box, either pick up or delivery

Reply:
Okay, cool